client/boot/index.js

```diff
@@ -172,12 +173,27 @@ function boot() {
 		if ( config.isEnabled( 'oauth' ) ) {
 			LoggedOutLayout = require( 'layout/logged-out-oauth' );
 		} else {
-			LoggedOutLayout = require( 'layout/logged-out' );
+			LoggedOutLayout = require( 'layout/themes-logged-out' );
```

Comment:
Since `themes-logged-out` is a 'pure' render tree, we can't render into sections directly in our controllers. How do we update our routing & controllers to take this into account, and how do we make this generically usable?